dietpi/misc/dietpi-nordvpn

```diff
@@ -333,6 +334,13 @@ _EOF_
 
 				Update_Wan_Ip
 
+			elif [[ $G_WHIP_RETURNED_VALUE == 'Update' ]]; then
+
+				G_EXEC cd /tmp/$G_PROGRAM_NAME
+				G_EXEC curl -sSfL https://downloads.nordcdn.com/configs/archives/servers/ovpn.zip -o 171.zip
+				G_EXEC unzip -o 171.zip -d/etc/openvpn
```

Comment:
Maybe remove `/etc/openvpn/ovpn_tcp` and `/etc/openvpn/ovpn_udp` to also remove possibly obsolete/removed servers from the list? Or too risky if admins place custom configs inside?

Reply:
Good idea:
```
G_EXEC_DESC='Removing old NordVPN configs' G_EXEC rm -Rf /etc/openvpn/ovpn_{tcp,udp}/*.nordvpn.com.{tcp,udp}.ovpn
```
The problem is that `*` will expand to every single file being added to the `rm` command, so that output is veeeery long 😄. Hence the G_EXEC_DESC to shorten the output.